api/envoy/config/core/v3/base.proto

```diff
@@ -444,3 +444,10 @@ message ControlPlane {
   // the Envoy is connected to.
   string identifier = 1;
 }
+
+// Configuration of DNS Lookup option flags.
+message AreaDnsLookupOptionFlags {
+  google.protobuf.BoolValue use_tcp = 1;
```

Comment:
`s/use_tcp/use_tcp_for_dns_lookups/g`

I suggest investigating why the existing field is deprecated so that we aren't introducing work that will potentially get removed.

Reply:
Included the deprecation note which appear on the docs. Please suggest any addition comments if needed, which can benefit the reader.